modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GeneTreeHighlighting_conf.pm

```diff
@@ -45,6 +45,10 @@ sub default_options {
     run_all      => 0,
     antispecies  => [],
     meta_filters => {},
+    production_db => 'ensembl_production',
+    production_host => 'mysql-ens-meta-prod-1-ensprod',
```

Comment:
I don't think this needs write access? Probably best to minimise how much info we provide about our servers and drop the 'ensprod'? In fact, could go a step further and just have `meta1`?

Reply:
Well spotted @james-monkeyshines, I got it wrong. production host can be ro but compara_host need to br rw. I've updated the code and use short names too.